### PR TITLE
Add support for debian-based packages

### PR DESCRIPTION
**Pull request by fridex**

### CODE REVIEW

**Comment by fridex** [comment]
Ready for a review.

File: thoth/package_extract/image.py [hunk 1]
```diff
@@ -32,7 +33,7 @@
 from .exceptions import NotSupported
 from .rpmlib import parse_nvra
 
-_LOGGER = logging.getLogger(__name__)
+_LOGGER = logging.getLogger('thoth.package_extract')
```
Comment (fridex) [inline thoth/package_extract/image.py]: This change shouldn't be there.

**Comment by fridex** [comment]
recheck

**Comment by fridex** [comment]
recheck

**Comment by fridex** [comment]
I will wait for @goern :)

**Comment by goern** [comment]
recheck

**Pull request merged**